package.json

```diff
@@ -4,13 +4,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon app.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "basic-auth": "^2.0.1",
-    "bcrypt": "^3.0.2",
```

Comment:
Why you change library version

Reply:
This version is not compatible with windows